modules/rules_foreign_cc/0.11.1/presubmit.yml

```diff
@@ -0,0 +1,10 @@
+matrix:
+  platform: ["centos7", "debian10", "macos", "ubuntu2004", "windows"]
+  bazel: ["6.x", "7.x"]
+
+tasks:
+  verify_targets:
+    name: "Verify build targets"
+    platform: ${{ platform }}
```

Comment:
Also "bazel: ${{ bazel }}" here.

Reply:
We could really do with proper docs for the format of presubmit.yml - there's not a definitive schema definition anywhere as far as I'm aware? 
Also this is probably something that should be caught by the parser - defining matrix values that aren't actually used in the presubmit?

Reply:
Yes, if you run `python3 tools/bcr_validation.py --check=rules_foreign_cc@0.11.1`, it should catch this error. I guess we could make it more explicit here: https://github.com/bazelbuild/bazel-central-registry/blob/main/docs/README.md#presubmit

Reply:
Yes, but the problem is that this file is defined at a distance and only made into the BCR via the BCR app so its tricky to validate this within the source of truth for this file.
I've opened an issue about providing full docs for presubmit.yml as its not documented fully; there are only examples of use in the current docs, not a full schema definition.

Reply:
The task config syntax follows [Bazel CI's specifications](https://github.com/bazelbuild/continuous-integration/tree/master/buildkite#configuring-a-pipeline) and I sent https://github.com/bazelbuild/bazel-central-registry/pull/2366

Reply:
Yes but that link is examples of usage only its not a full schema spec for the yaml format.